integrations/db-filedb/src/FileDb.ts

```diff
@@ -16,26 +8,16 @@ export interface FileDbConfig extends DbPluginConfig {
   primaryKeyColumn?: string;
 }
 
-export interface FileDbItem {
-  id: string;
-  user?: PersistableUserData;
-  session?: PersistableSessionData;
-  createdAt?: string;
-  updatedAt?: string;
-}
+export class FileDb extends DbPlugin<FileDbConfig> {
+  constructor(config?: FileDbConfig) {
+    super(config);
+    console.log(this.config);
+  }
 
-export class FileDb extends Plugin<FileDbConfig> {
   getDefaultConfig(): FileDbConfig {
     return {
+      // ...super.getDefaultConfig(),
```

Comment:
Is this intentionally disabled? Wouldn't this cause the config to not contain the default-config of `DbPlugin` because it never gets called?

Reply:
This was a WIP commit. The double slash is removed